app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -435,6 +459,24 @@ class HomeActivityTest {
     }
   }
 
+  /**
```

Comment:
```suggestion
  /*
```

Should just be a block comment, not a KDoc for the test.